doc/classes/Line2D.xml

```diff
@@ -54,6 +54,7 @@
 			<return type="int">
 			</return>
 			<description>
+				Return the amount of points in the Line2D.
```

Comment:
The method does the action:
`Return` -> `Returns`
Use [code] markup for Classes etc. you don't want to link to, and [ClassName] to generate a link.
`[code]Line2D[/code]